auth-web/src/router.ts

```diff
@@ -37,6 +38,7 @@ const routes = [
   { path: '/returnpayment/:paymentId/transaction/:transactionId', component: PaymentReturnForm, props: mapReturnPayVars, meta: { requiresAuth: true } },
   { path: '/createaccount', component: CreateAccount, meta: { requiresAuth: false } },
   { path: '/dashboard', component: Dashboard, meta: { requiresAuth: true } },
+  { path: '/addbusiness', component: AddBusinessForm },
```

Comment:
You'll need the `requiresAuth` meta property here.  See the other routes for an example.